playground/src/client/index.ts

```diff
@@ -16,15 +16,23 @@
 
 import puppeteer, {Browser} from 'puppeteer';
 import {Page} from 'puppeteer';
+import {EmulationOptions, NOT_EMULATED} from './emulationOptions';
 import {
   clickSearchResultLink,
   setupObserver,
   getObserverResult,
 } from './evaluated';
 
-async function setupPage(page: Page) {
-  await page.emulate(puppeteer.devices['Pixel 5']!);
-  await page.emulateNetworkConditions(puppeteer.networkConditions['Fast 3G']!);
+async function setupPage(page: Page, emulationOptions: EmulationOptions) {
+  const {device, networkCondition} = emulationOptions;
+  if (device !== NOT_EMULATED) {
```

Comment:
Do  you have any unit test setup for when device doesn't default to NON_EMULATED?

Reply:
The integration test uses the default cli params, with device set to `Pixel 5`. The default value of cli param is declared in `playground/src/index.ts`.